docs/getting_started/setup_android.md

```diff
@@ -0,0 +1,50 @@
+Setting up Latest Web3J Library for Android Development
```

Comment:
should be Web3j

Reply:
Also change the title here as it is not a blog post, maybe something like - "Setting up project for Android"